cfme/tests/networks/test_ovn.py

```diff
@@ -0,0 +1,67 @@
+import fauxfactory
```

Comment:
**OPTIONAL:** Might be good to have doc string that explains what this is supposed to test, and defines the ovn acronym to mean Ovirt Virtual Network, with maybe a link to where these things are described.